aws/environments/latest.yml

```diff
@@ -12,29 +12,3 @@ owner: "dev-fxacct@mozilla.org"
 reaper_spare_me: "true"
 
 content_static_resource_url: https://static-latest.dev.lcip.org
-
-# N.B. the leading newline on the value below is important,
-# it prevents ansible from parsing it as a python dict.
-content_scoped_keys_validation: >-
```

Comment:
I think this was using a custom config during the scoped-keys rollout, but I don't think there's any need for it any longer.  I've merged it with the default config below.